server.js

```diff
@@ -1,7 +1,30 @@
-var express = require('express');
-var app = express();
-app.get('/ping', (req, res) =>{
-    res.send('hello ')
-})
-
-var server = app.listen(3000);
+const express = require("express");
+const app = express();
+const mongoose = require('mongoose');
+
+const uri = "mongodb+srv://hanshul:babaji@cluster0.e06nfsc.mongodb.net/?retryWrites=true&w=majority";

```

Comment:
The MongoDB connection string is hardcoded in the code, which is not a good practice. It's better to use environment variables for sensitive data like this. [important]